torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -199,12 +216,19 @@ void IRPrinter::handle(const Half* const h) {
   }
 }
 
-void IRPrinter::handle(const Int* const i) {
-  if (print_inline_ && FusionGuard::getCurFusion()->origin(i) != nullptr) {
-    os << "( ";
-    handle(FusionGuard::getCurFusion()->origin(i));
-    os << " )";
-    return;
+void IRPrinter::handle(const Int* i) {
+  // Make sure we didn't bypass the value mapping
+  // (for example calling IRPrinter::handle() with a Int*)
```

Comment:
Will we have to add this to every all other types?

Reply:
Right now we only map Ints (sizes), so this would be the most likely point of failure. I didn't want to spray this paranoid check everywhere, although I could be easily be convinced to do so.

Reply:
Given that this is a stopgap, yes, that makes sense.